crowdin.yml

```diff
@@ -50,3 +50,7 @@ files:
   - source: /jekyll/_cci2/images/linux-vm/*.md
     translation: /jekyll/_cci2_%two_letters_code%/images/linux-vm/%original_file_name%
     update_option: update_as_unapproved
+    
+  - source: /jekyll/_cci2/server/v4.1/air-gapped-installation/*.adoc
```

Comment:
I think we actually need to add a few directories here: 
/server/v4.1/
/server/v4.1/air-gapped-installation/
/server/v4.1/installation/
/server/v4.1/overview/
/server/v4.1/operator/